.github/workflows/minimal-app.yml

```diff
@@ -0,0 +1,93 @@
+name: minimal-app-workflow
+
+on:
+  workflow_dispatch:
+  pull_request:
+    paths:
+      - minimal-app/**
+      - .github/workflows/minimal-app.yml
+  push:
+    branches:
+      - main
+    paths:
+      - operator/**
+      - "!operator/webhook/**"
+      - .github/workflows/operator.yml
+
+env:
+  WORKING_DIR: ./minimal-app
+  REGISTRY: ghcr.io/octoconsulting
+  IMAGE_NAME: keip/minimal-app
```

Comment:
:question: Is there anyway we can sync the `REGISTRY` and `IMAGE_NAME` with `minimal-app`'s pom.xml?

Reply:
Updated to grab registry and image-name from the pom.xml.